server/src/test/java/io/spine/server/SubscriptionServiceTest.java

```diff
@@ -206,6 +206,7 @@ private void checkSubscribesTo(Class<? extends Message> aClass) {
     }
 
     @Test
+    @MuteLogging
     @DisplayName("not subscribe to a system event")
     void notSubscribeToSystemEvent() {
```

Comment:
This test also does not seem to throw "expected" exceptions, so I don't know why mute.